src/promptflow/sample_trace.json

```diff
@@ -0,0 +1,290 @@
+[
```

Comment:
I do not see samples for LLM calls. Can you please add some and UX can start working on the LLM specific visualization
![image](https://github.com/microsoft/promptflow/assets/5934735/b4d29cda-e836-4ced-be23-ef310edaa117)